rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -133,7 +133,7 @@ namespace vk
 
 		// v3dv and PanVK support BC1-BC3 which is all we require, support is reported as false since not all formats are supported
 		optional_features_support.texture_compression_bc = features.textureCompressionBC
```

Comment:
Is the flag set for v3dv?
Shouldn't it be false anyway for those. I'm starting to question why this check exists like this.
I believe panvk/v3dv support BC but cannot set the feature flag which means previous impl was correct.